pkg/querier/worker/worker.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 // WorkerConfig is config for a worker.
+// nolint:golint
```

Comment:
Instead of ignoring the lint, could we just rename it to `Config`?

Reply:
This is done in next PR, which is why it's just ignored here.